feature-detects/es6/symbol.js

```diff
@@ -0,0 +1,34 @@
+/*!
+{
+  "name": "ES6 Symbol",
+  "property": "es6symbol",
+  "notes": [{
+    "name": "Official ECMAScript 6 specification",
+    "href": "https://www.ecma-international.org/ecma-262/6.0/#sec-symbol-constructor"
+  }],
+  "polyfills": ["es6shim"],
+  "authors": ["Ron Waldon (@jokeyrhyme)"],
```

Comment:
you can put your name here :-)

Reply:
I thought here are names of the authors of polyfills, and I'm not one.

Reply:
Changed.